docs/cheat_sheet.md

```diff
@@ -30,12 +30,15 @@ When developing, these commands may come in handy:
 
 | Argument    | Description|
 | -------- | ------- |
+| `make install`  |  Install dependencies and activates the virtual environment. |
 | `make run`  |  Runs `server.py` |
 | `make run_docker`  |  Runs `docker compose up -d` |
 | `make test`  |  Runs pytest |
 | `make test_docker`  |  Runs pytest on Docker |
 | `make output_coverage`  |  Outputs the coverage of the tests |
 | `make send_email`  |  Runs `send_email.py` |
+| `make lint`  |  Runs the ruff linter |
+| `make format`  |  Runs the ruff formatter |
```

Comment:
It's a minor point, but is `make clean` unnecessary?

Reply:
@K-dash ah, I forgot to add that